servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -932,13 +932,19 @@ void RenderForwardMobile::_render_scene(RenderDataRD *p_render_data, const Color
 		{
 			// regular forward for now
 			Vector<Color> c;
-			c.push_back(clear_color.srgb_to_linear()); // our render buffer
-			if (rb_data.is_valid()) {
-				if (p_render_data->render_buffers->get_msaa_3d() != RS::VIEWPORT_MSAA_DISABLED) {
-					c.push_back(clear_color.srgb_to_linear()); // our resolve buffer
+			{
+				Color cc = clear_color.srgb_to_linear();
+				if (rb_data.is_valid()) {
+					cc.a = 0; // for transparent viewport backgrounds
 				}
-				if (using_subpass_post_process) {
-					c.push_back(Color()); // our 2D buffer we're copying into
+				c.push_back(cc); // our render buffer
```

Comment:
```suggestion
				c.push_back(cc); // Our render buffer.
```
As per the [style guide](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide)

Reply:
All these comments (other than the "for transparent viewport backgrounds" one) were in the code already. They were only marked as new / changed because I increased the indentation. I went ahead and addressed all of them in a force-push, though, since I'm in the area anyways.

Reply:
Most comments in the codebase pre-date the style guide. So we are making an effort to make sure that they get updated when nearby code is changed. That way we get consistent style without muddying our git history too much.